test/Serializer/ArticleVoRNormalizerTest.php

```diff
@@ -276,6 +276,46 @@ public function normalizeProvider() : array
                             'filename' => 'image.jpeg',
                         ],
                     ],
+                    'dataSets' => [
+                        'generated' => [
+                            [
+                                'id' => 'id',
+                                'date' => '2000-01-02',
+                                'authors' => [
+                                    [
+                                        'type' => 'person',
+                                        'name' => [
+                                            'preferred' => 'preferred name',
+                                            'index' => 'index name',
+                                        ],
+                                    ],
+                                ],
+                                'title' => 'title',
+                                'uri' => 'https://doi.org/10.1000/182',
+                                'authorsEtAl' => true,
+                                'dataId' => 'data id',
+                                'details' => 'details',
+                                'doi' => '10.1000/182',
+                            ],
+                        ],
+                        'used' => [
+                            [
+                                'id' => 'id',
+                                'date' => '2000',
+                                'authors' => [
+                                    [
+                                        'type' => 'person',
+                                        'name' => [
+                                            'preferred' => 'preferred name',
+                                            'index' => 'index name',
+                                        ],
+                                    ],
+                                ],
+                                'title' => 'title',
+                                'uri' => 'http://www.example.com/',
+                            ],
+                        ],
+                    ],
```

Comment:
loks asymmetric, a couple of `->withGeneratedDataSets(...)->withUsedDataSets(...)` for this example would make it consistent